### PR TITLE
TCGABiolinks R/Bioconductor package

### PR DESCRIPTION
**Pull request by fabio-cumbo**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

The aim of TCGAbiolinks is : i) facilitate the GDC open-access data retrieval, ii) prepare the data using the appropriate pre-processing strategies, iii) provide the means to carry out different standard analyses and iv) to easily reproduce earlier research results. In more detail, the package provides multiple methods for analysis (e.g., differential expression analysis, identifying differentially methylated regions) and methods for visualization (e.g., survival plots, volcano plots, starburst plots) in order to easily develop complete analysis pipelines.

### CODE REVIEW

File: recipes/bioconductor-tcgabiolinks/build.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+#
+$R CMD INSTALL --build .
+#
+# # Add more build steps here, if they are necessary.
+#
+# See
+# http://docs.continuum.io/conda/build.html
+# for a list of environment variables that are set during the build
+# process.
+#
```
Comment (daler) [inline recipes/bioconductor-tcgabiolinks/build.sh]: rm extra comments from here please?

**Comment by lecorguille** [comment]
[`bioconductor-consensusclusterplus`](https://github.com/bioconda/bioconda-recipes/pull/5400) will be soon available through conda but you also need to create a package for [`r-survminer`](https://travis-ci.org/bioconda/bioconda-recipes/jobs/259849888#L814) before building this one

**Comment by fabio-cumbo** [comment]
Thanks for your review @lecorguille! I have created a PR on conda-forge about the r-survminer package [#3404](https://github.com/conda-forge/staged-recipes/pull/3404)

**Comment by daler** [comment]
@lecorguille in #5398 I had asked @fabio-cumbo to add `r-survminer` to conda-forge since it was not a bioinformatics-specific package. Same with #5395, #5396, #5397.

@fabio-cumbo, thanks for your contributions, and thanks for opening the conda-forge PR!

**Comment by daler** [comment]
@fabio-cumbo I bundled your changes in #5401, #5402, #5403, and #5404 together into #5478 so hopefully testing and merging will be more straightforward. Closing this for now . . .

**Pull request closed without merging**